sfputil/main.py

```diff
@@ -894,6 +950,10 @@ def set_lpmode(logical_port, enable):
         click.echo("Error: No physical ports found for logical port '{}'".format(logical_port))
         return
 
+    if is_rj45_port_from_api(logical_port):
```

Comment:
why adding it to the cli itself instead of calling the api and it will provide the proper implementation or maybe even error in the log?

Reply:
see the discussion here: https://github.com/keboliu/sonic-utilities/pull/10/files#r815409137
API return code can not give out enough msg, need to handle it in CLI